doc/sphinx/source/data/data-config.rst

```diff
@@ -47,7 +47,7 @@ The following lines will check whether a newly added theory can be read by valid
 ..  code-block:: python
     
     from nnpdf_data import theory_cards
-    from validphys.theorydbutils import fetch_theory
+    from nnpdf_data.theorydbutils import fetch_theory
     theory = fetch_theory(theory_cards, 700)
```

Comment:
The full vp path is also mentioned a few lines above

Reply:
My sed-fu failed me :'(

thanks!